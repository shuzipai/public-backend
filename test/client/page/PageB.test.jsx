import React from 'react'
import renderer from 'react-test-renderer'

import PageB from '../../../client/page/PageB'

test('<PageB> html', () => {
  const component = renderer.create(
    <PageB></PageB>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
