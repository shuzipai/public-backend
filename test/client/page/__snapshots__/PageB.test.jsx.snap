// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageB> html 1`] = `
<div
  className="try"
>
  页面B
</div>
`;
